src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 15px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
Why do you need it?

I guess you can remove it and increase max-width for container or just use sass function and calculate max-width

Reply:
if I understand correctly, the test requires a max-width of 1024px and if there are less than 8 elements, then additional properties will need to be set for the second row

Reply:
yeah, but now it looks strange on screens smaller then 1024px
![image](https://user-images.githubusercontent.com/80700247/222735037-2adcfdf7-6864-4da5-9fe1-ab6a71dfb22f.png)